app/src/main/java/com/github/se/cyrcle/ui/profile/ProfileScreen.kt

```diff
@@ -26,6 +26,12 @@ import androidx.compose.ui.unit.dp
 import coil.compose.rememberAsyncImagePainter
 import coil.request.ImageRequest
 import coil.transform.CircleCropTransformation
+import com.github.se.cyrcle.ui.card.Location
+import com.github.se.cyrcle.ui.card.Parking
```

Comment:
The paths for those imports are weird , it should come from the model not another Screen

Reply:
Done